mitmproxy/command.py

```diff
@@ -100,8 +100,10 @@ def signature_help(self) -> str:
     def prepare_args(self, args: typing.Sequence[str]) -> inspect.BoundArguments:
         try:
             bound_arguments = self.signature.bind(*args)
-        except TypeError as v:
-            raise exceptions.CommandError(f"Command argument mismatch: {v.args[0]}")
+        except TypeError:
+            expected = f'Expected: {str(self.signature.parameters)}'
+            received = f'Received: {str(args)}'
+            raise exceptions.CommandError(f"Command argument mismatch: \n\t{expected}\n\t{received}")
```

Comment:
What is this solving here? IIRC `v.args[0]` was relatively sensible.

Reply:
v.args[0] doesn't give you any information about the args that were expected and what args were received. This facilitates debuging things if an error like that occurs.

Reply:
I'll get a proper `missing a required argument: 'y'` (as you would in the Python console).

Reply:
Here is what I get if I give something too many parameters:

```
mitmproxy.exceptions.CommandError: Command argument mismatch: too many positional arguments
```

Doesn't tell me anything about how many extras I have given or what is expected.